dev/values/validators/all.yml

```diff
@@ -36,6 +36,9 @@ genesis:
     # address of the bridge account. needs funds to sign bridge init tx
     - address: astria1d7zjjljc0dsmxa545xkpwxym86g8uvvwhtezcr
       balance: "69000000"
+    # address of the auctioneer account. needs funds to submit auction results.
+    - address: astria1l5hg09ak99cfua5gu33j2m2se8eunv8nazyxtr
+      balance: "69000420"
```

Comment:
Should make sure dev private key is known, preferably by adding to default `dev/values/auctioneer/values.yaml`

Reply:
it's in `dev/values/auctioneer/values.yaml` here: https://github.com/astriaorg/astria/blob/9aaf7527acf40bf9bbbabe481085206fd58f895a/dev/values/auctioneer/values.yaml#L12